src/components/ChainInput/chainInput.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { CHAINS, ChainId } from '@pangolindex/sdk';
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { ChainInputProps } from './types';
+import { ChainInput } from '.';
+
+export default {
+  component: ChainInput,
+  title: 'Components/ChainInputs',
+  parameters: {
+    docs: {
+      description: {
+        component: 'This is the main component of the bridge page.',
+      },
+    },
+  },
+  argTypes: {
+    chain: {
+      name: 'Chain',
+      control: 'object',
+      type: { name: 'object', required: false },
+      description: 'Chain',
+    },
+    buttonStyle: {
+      name: 'Chain',
+      control: 'object',
+      type: { name: 'object', required: false },
+      description: 'Button Styles',
+    },
+    onChainClick: {
+      name: 'onChainClick',
+      control: 'function',
+      type: { name: 'function', required: false },
+      description: 'It triggers when the chain is clicked',
+    },
+  },
+};
+
+const TemplateChainInput: ComponentStory<typeof ChainInput> = (args: any) => <ChainInput {...args} />;
```

Comment:
Its not proper looking in storybook, i think it has less padding ?

<img width="550" alt="Screenshot 2022-11-14 at 11 43 58 PM" src="https://user-images.githubusercontent.com/6604146/201735277-e2937ed9-4a66-488f-ad2d-5cb132c45aab.png">

Reply:
fixed